### PR TITLE
Initial implementation for ChromaTof feature table parsing.

### PR DESCRIPTION
**Pull request by nilshoffmann**
I have also added some of the test files that I used in Maltcms in gzip format (saves a lot of space in git).

### CODE REVIEW

File: msdk-datamodel/src/main/java/io/github/msdk/datamodel/impl/converter/CopyConverter.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301  USA
+ */
```
Comment (dyrlund) [inline msdk-datamodel/src/main/java/io/github/msdk/datamodel/impl/converter/CopyConverter.java]: Shouldn't this be the MSDK license?

File: msdk-datamodel/src/main/java/io/github/msdk/datamodel/impl/converter/IonAnnotationConverter.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301  USA
+ */
```
Comment (dyrlund) [inline msdk-datamodel/src/main/java/io/github/msdk/datamodel/impl/converter/IonAnnotationConverter.java]: Shouldn't this be the MSDK license?

**Comment by tomas-pluskal** [comment]
Thanks, Nils!
I noticed your ParserUtilities class fails to compile under Java 7, because it imports javafx.util.Pair.
Also, please update the license as Thomas pointed out above.
Cheers.

**Comment by nilshoffmann** [comment]
PR https://github.com/msdk/msdk/pull/92 should resolve these issues.

**Pull request merged**